packages/devtools_app/test/screens/debugger/debugger_screen_test.dart

```diff
@@ -172,6 +172,26 @@ void main() {
     },
   );
 
+  testWidgetsWithWindowSize(
+    'debugger exception mode tooltip',
+    smallWindowSize,
+    (WidgetTester tester) async {
+      await tester.pumpWidget(
+        wrapWithControllers(
+          Builder(builder: screen.build),
+          debugger: debuggerController,
+        ),
+      );
+      expect(
+        _findDevToolsTooltip(
```

Comment:
you can use find.byTooltip instead of creating this helper method

Reply:
Ha! You're totally right. Initially I though that didn't work and convinced myself we needed to match on `DevToolsTooltip`s specifically but `byTooltip` works as-is. Thanks!